.github/workflows/integration.yaml

```diff
@@ -9,22 +9,22 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        k8s_version: [v1.18.2, v1.17.5, v1.16.9]
+        k8s_version: [v1.20.0, v1.19.4, v1.18.8, v1.17.11, v1.16.15]
```

Comment:
We want to test against only the three latest versions of k8s, so keep the `v1.20.0`, `v1.19.4` and `v1.18.8`

Reply:
README says "Kubernetes cluster 1.16+" :wink:

I think it would be better to have another issue/PR for dropping 1.17 and 1.16.

Reply:
sure